### PR TITLE
Pegout creation index fixes for security review

### PR DESCRIPTION
**Pull request by nathanieliov**

### CODE REVIEW

**Comment by nathanieliov** [comment]
Pipeline:run

**Comment by nathanieliov** [comment]
pipeline:run

**Comment by nathanieliov** [comment]
pipeline:run

File: rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java [hunk 1]
```diff
@@ -1094,44 +1094,40 @@ public void serializeSha256Hash_zeroHash() {
         );
     }
 
-    @Test
+    @Test(expected = NullPointerException.class)
     public void serializeSha256Hash_nullValue() {
-        test_serializeSha256Hash(null, new byte[]{});
+        test_serializeSha256Hash(null, null);
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java:1099]: Shouldn't we keep the test case where empty byte array is sent?

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**